### PR TITLE
Fix #1194 build variants

### LINKED ISSUE #1194: #API variant
API 'variant.getPreBuild()' is obsolete and has been replaced with 'variant.getPreBuildProvider()'.
It will be removed at the end of 2019.
For more information, see https://d.android.com/r/tools/task-configuration-avoidance.
To determine what is calling variant.getPreBuild(), use -Pandroid.debug.obsoleteApi=true on the command line to display a stack trace.

### PR DESCRIPTION
**Pull request by elirehema**
Fixes #1194 

Please Add Screenshots If there are any UI changes.

Please make sure these boxes are checked before submitting your pull request - thanks!

- [x] Apply the `AndroidStyle.xml` style template to your code in Android Studio.

- [x] Run the unit tests with `./gradlew check` to make sure you didn't break anything

- [x] If you have multiple commits please combine them into one commit by squashing them.

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -47,7 +47,7 @@ ext {
     playServicesVersion = '16.1.0'
     firebaseMessagingVersion = '17.6.0'
     oss_licenses = '16.0.2'
-    kotlinVersion = '1.3.11'
+    kotlinVersion = '1.3.40'
```
Comment (geekanamika) [inline build.gradle]: Since, we're updating kotlin version, let's make it most updated & stable version i.e 1.3.60?

**Review by geekanamika — CHANGES_REQUESTED** [review]
Add a single comment, please update PR.

File: build.gradle (hunk 1 above)
Comment (elirehema) [reply build.gradle]: @geekanamika i have updated the Kotlin Version to last version

**Comment by elirehema** [comment]
> Add a single comment, please update PR.

Okay its almost DONE

**Pull request closed without merging**